### ISSUE TITLE
Changes to lacework_integration_ecr Name field have no effect

### ISSUE THREAD

**Issue by alexeiser** [lacework/terraform-provider-lacework#85]
If a Lacework ECR resource has been created, or imported - and the name value changes no effect is shown on the plan (as if ignore_changes was set).

Example resource config, with a changed name.
```
resource "lacework_integration_ecr" "lacework-ecr-integration" {
  name              = "new name"
  registry_domain   = "${data.aws_caller_identity.current.account_id}.dkr.ecr.${var.region}.amazonaws.com"
  access_key_id     = aws_iam_access_key.lacework-ecr-access_key.id
  secret_access_key = aws_iam_access_key.lacework-ecr-access_key.secret
}
```

Existing object in Lacework:
```
 {
      "INTG_GUID": "GRAILBIO_XXXXX",
      "NAME": "lacework-ecr-integration",
      "CREATED_OR_UPDATED_TIME": "2020-Dec-11 19:48:39 UTC",
      "CREATED_OR_UPDATED_BY": "XXXX",
      "TYPE": "CONT_VULN_CFG",
      "ENABLED": 1,
      "STATE": {
        "ok": true,
        "lastUpdatedTime": "2021-Mar-09 03:16:50 UTC",
        "lastSuccessfulTime": "2021-Mar-09 03:16:50 UTC"
      },
      "IS_ORG": 0,
      "DATA": {
        "ACCESS_KEY_CREDENTIALS": {
          "ACCESS_KEY_ID": "XXXXXXXXX"
        },
        "AWS_AUTH_TYPE": "AWS_ACCESS_KEY",
        "REGISTRY_TYPE": "AWS_ECR",
        "REGISTRY_DOMAIN": "XXXXXXXX.dkr.ecr.us-west-2.amazonaws.com",
        "LIMIT_BY_TAG": "*",
        "LIMIT_BY_LABEL": "*",
        "LIMIT_BY_REP": "",
        "LIMIT_NUM_IMG": 5
      },
      "TYPE_NAME": "CONT_VULN_CFG"
    },
```

Terraform plan result
```

No changes. Infrastructure is up-to-date.
```

**Comment by afiune** [comment]
Hi @alexeiser 👋🏽!! Thank you for your feedback, we are going to try to replicate this bug in our environment, could you please help us by providing the version of Terraform you are running on?

**Comment by afiune** [comment]
Tested on Terraform 14 and seems to be working fine:
<img width="1279" alt="Screen Shot 2021-03-08 at 11 36 29 PM" src="https://user-images.githubusercontent.com/5712253/110423693-c1b3cf80-805e-11eb-99ef-883757990fae.png">

I will work my way down to Terraform 12 tomorrow morning.

**Comment by alexeiser** [comment]
Yes - we are using TF 0.12

**Comment by afiune** [comment]
@alexeiser Gotcha! Could you share the complete version?

We support `0.12.26` and above:
- https://support.lacework.com/hc/en-us/articles/360061585353-Terraform-for-Lacework-Overview

**Comment by alexeiser** [comment]
We are using 0.12.29. 

Switching temporarily to 
Terraform v0.13.5
+ provider registry.terraform.io/-/lacework v0.2.14

Does not change the behavior.

Turns out - that resource targeting is the cause. When I removed resource targeting all the name changes are shown.

When reverting to 0.12.29 and planning with out targeting it does work.

**So - this is really a bug report that targeting resources doesn't work.**
